scripts/ci-e2e-job-trigger-checker.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Usage:
+#   ./scripts/ci/ci-e2e-job-trigger-checker.sh
+#
+# This script will query information about the current PR when used in CircleCI environment.
+# The information is used to decide whether the end-to-end compatibility testing pipeline needs to be launched.
+#
+
+if [ "$CIRCLE_TAG" != "" ]; then
+    echo "We are on tag $CIRCLE_TAG, trigger all the bots."
+    exit 0
+fi
+
+export GITHUB_TOKEN=$(mbx-ci github reader token)
+
+IS_BRANCH_PROTECTED=$(gh api repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/branches/$CIRCLE_BRANCH --jq .protected)
+if [ $IS_BRANCH_PROTECTED != "false" ]; then
+    echo "We are on protected branch, trigger all the bots."
+    exit 0
+fi
+
+if [ -z "$CIRCLE_PULL_REQUEST" ]; then
+    echo "No pull request created yet. Please create pull request in order to finish CI."
+    exit 1
+fi
+
+echo "Checking PR $CIRCLE_PULL_REQUEST"
+
+COMMIT_MESSAGE=$(gh api repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/commits/$CIRCLE_SHA1 --jq .commit.message)
+#echo $COMMIT_MESSAGE
+
+if [ -z "${COMMIT_MESSAGE##*"run_e2e"*}"  ]; then
```

Comment:
now we should add `run_e2e` to commit message? 
will you add this to our readme?

Reply:
It can be used as a shortcut to trigger the e2e test with default versions without adding labels.

However, this is optional and can be removed. It originates from gl-native, where this approach is used for other purposes.